src/common/const.go

```diff
@@ -105,7 +105,6 @@ const (
 	ClairURL                          = "clair_url"
 	NotaryURL                         = "notary_url"
 	DefaultAdminserverEndpoint        = "http://adminserver:8080"
-	DefaultJobserviceEndpoint         = "http://jobservice:8080"
```

Comment:
Why this is removed? Did you mean to delete the `DefaultAdminserverEndpoint `?

Reply:
All config default value should be kept in metadatalist.go